gears/src/types/tx/mod.rs

```diff
@@ -26,6 +29,53 @@ pub trait TxMessage:
     fn type_url(&self) -> &'static str;
 }
 
+/// Utility type that guarantees correctness of transaction messages set
+pub struct Messages<T: TxMessage> {
+    messages: Vec<T>,
+    /// A number of messages in the transaction. Zero means unlimited number of messages.
+    /// Default is 0
+    chunk_size: u16,
```

Comment:
Does it make sense to make this a usize from the get go?

Reply:
I think yes. Thank you.